nuntium/templates/nuntium/message/post_submission.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<div class="jumbotron">
+    <div class="container">
+        <h1>{% trans "Your message has been created!" %}</h1>
+        {% if messages %}
```

Comment:
I agree. The fact that the message creator needs to check their email and click the confirmation link is the reason why I think we need a separate page - I was worried that that info might get lost and not acted upon as just a message on the instance homepage. It really needs to be in the main body of the page that you get to after sending a message.